nemo/collections/tts/models/fastspeech2_hifigan_e2e.py

```diff
@@ -54,7 +54,7 @@ def __init__(self, cfg: DictConfig, trainer: 'Trainer' = None):
             cfg = OmegaConf.create(cfg)
         super().__init__(cfg=cfg, trainer=trainer)
 
-        self.audio_to_melspec_preprocessor = instantiate(cfg.preprocessor)
+        self.audio_to_melspec_precessor = instantiate(cfg.preprocessor)
```

Comment:
Is it possible to put this in a try block?

Reply:
We have this typo in several models. Let’s discuss how to resolve it out of this PR. In my opinion, code will be too hard with try block, because we can't check only config field, we would need to support two variants of attribute of class (because torch load weights function looks for concrete attribute of class).